<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userManagement_Dialog</class>
 <widget class="QDialog" name="userManagement_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户管理 - 求是图书管理系统</string>
  </property>
  <widget class="QGroupBox" name="group_newuser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>添加新用户</string>
   </property>
   <widget class="QPushButton" name="newuser_add_Button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>创建</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newuser_stuid">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>学号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newuser_password">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>部门：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newuser_department">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="newuser_num_limit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>借书上限：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newuser_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="newuser_isadmin">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>普通用户</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>管理员</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_user">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>用户管理</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeView" name="userview">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated">
       <bool>false</bool>
      </property>
      <property name="headerHidden">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="group_manageuser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>571</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>管理用户</string>
   </property>
   <widget class="QPushButton" name="manageuser_modify_Button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manage_stuid">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>学号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manage_password">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>部门：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manage_department">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="manage_num_limit">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>借书上限：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manage_name">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="manage_isadmin">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>普通用户</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>管理员</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="manageuser_delete_Button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_nav">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>操作选项</string>
   </property>
   <widget class="QPushButton" name="guide_newuser_Button">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>添加用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="guide_user_Button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>用户管理</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
